### DISCUSSION TITLE
DLC: S.O.G. Prairie Fire - mission BTC  ??

### DISCUSSION THREAD

**Discussion by ocibisi** [Vdauphin/HeartsAndMinds#1489] [Question and answer]
Hello everyone, sorry for my school English... I hope you can understand me.
With my group we are trying to play the BTC mission by putting it on the server. Using Arma 3 SOG PRAIRIE DLC but we are having problems with medical treatment.
Do you have problems too?
Problem example:
When you get hit you can't heal wounds and all human figure in ace medic menu is white.
Do you have any information about it?
Thank you

**Comment by Vdauphin** [comment]
Hello,

You should ask to ACE team or S.O.G team

Cheers

**Reply by ocibisi** [reply]
ok thanks
